<launch>

<node name="reggie_episode_node" pkg="reggie_episode" type="reggie_episode" >
  <param name="episode_directory" value="/home/stevie/reggie_ws/src/reggie/reggie_support/episodes" />
</node>

</launch>